### PR TITLE
Add LottieFiles

### PR DESCRIPTION
**Pull request by hkamran80**

### CODE REVIEW

File: img/l/lottiefiles.com.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 156 156"><path fill="#00ddb3" d="M116.5 0h-77A39.5 39.5 0 0 0 0 39.5v77A39.5 39.5 0 0 0 39.5 156h77a39.5 39.5 0 0 0 39.5-39.5v-77A39.5 39.5 0 0 0 116.5 0Z"/><path fill="#fff" d="M118.5 35C91.7 35 81.7 54 73.7 69.4l-5.2 9.9c-8.5 16.3-14.8 26.2-31 26.2A7.7 7.7 0 0 0 32 108a7.7 7.7 0 0 0 0 10.9 7.7 7.7 0 0 0 5.4 2.2c26.9 0 36.8-19.1 44.8-34.5l5.2-9.9c8.5-16.3 14.8-26.2 31.1-26.2A7.7 7.7 0 0 0 124 48a7.7 7.7 0 0 0 2.3-5.5 7.7 7.7 0 0 0-7.8-7.7z"/></svg>
```
Comment (Carlgo11) [inline img/l/lottiefiles.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 156 156"><rect width="156" height="156" fill="#00ddb3" ry="25.5"/><path fill="#fff" d="M118.5 34.8c-26.8 0-36.8 19.2-44.8 34.6l-5.2 9.9c-8.5 16.3-14.8 26.2-31 26.2-2.1 0-4 1-5.5 2.5-3 3-3 7.9 0 10.9a7.7 7.7 0 0 0 5.4 2.2c26.9 0 36.8-19.1 44.8-34.5l5.2-9.9c8.5-16.3 14.8-26.2 31.1-26.2 2.1 0 4-1 5.5-2.5a7.7 7.7 0 0 0-5.5-13.2z"/></svg>
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: img/l/lottiefiles.com.svg (hunk 1 above)
Comment (hkamran80) [reply img/l/lottiefiles.com.svg]: How did you generate this?

Comment (Carlgo11) [reply img/l/lottiefiles.com.svg]: Uhm, I removed the `fill="none"` manually. Opened the SVG in Inkscape and replaced the first path with a rectangle object. Then I just added the radius and saved the SVG.

**Review by Carlgo11 — APPROVED** [review]

File: img/l/lottiefiles.com.svg (hunk 1 above)
Comment (hkamran80) [reply img/l/lottiefiles.com.svg]: Ah. Thanks!

**Pull request merged**